modules/stereo/src/Descriptor.cpp

```diff
@@ -0,0 +1,448 @@
+//By downloading, copying, installing or using the software you agree to this license.
```

Comment:
As a general note, please, use consistent "white-space style" in all of your files. That is,

- Use only spaces for indentation (not tabs). 
- Default indentation width should equal to 4 spaces.
- Don't leave trailing white space.